static/js/publisher/pages/Releases/components/releasesConfirmDetails/releaseRow.js

```diff
@@ -22,39 +12,13 @@ const ReleaseRow = ({
         <b>{channel}</b> on <b>{revisionInfo.architectures.join(", ")}</b>
       </span>
     </span>
-    {progress && (
```

Comment:
So, we are not doing separate progress bar per revision? It's all grouped?

<img width="1144" alt="image" src="https://github.com/user-attachments/assets/e33aebde-a3bb-4813-802a-73f2fe05636f" />

Reply:
Yes, for this iteration they will be all grouped intentionally